common/models/product.json

```diff
@@ -19,8 +19,7 @@
       "url": "string"
     },
     "document": {
-      "apis": 
-        {},
+      "apis": "object",
```

Comment:
It was my understanding that if you wanted to allow _any_ object there, that you'd use `"object"`, but if you wanted to explicitly define the properties that make up that object, you'd have to have at least one key-value pair in the definition.